Extension/src/LanguageServer/extension.ts

```diff
@@ -912,6 +913,11 @@ function onLogDiagnostics(): void {
     clients.ActiveClient.logDiagnostics();
 }
 
+function onRescanFolder(): void {
+    onActivationEvent();
+    clients.ActiveClient.rescanFolder();
```

Comment:
need to think about multi-root

Reply:
I talked with @sherryyshi about this and we decided to go with "rescan workspace"